src/controllers/settings_controller.js

```diff
@@ -313,6 +314,32 @@ SettingsController.post('/system/actions', auth, (request, response) => {
   }
 });
 
+SettingsController.get('/system/ntp', (request, response) => {
+  const statusImplemented = Platform.implemented('getNtpStatus');
+
+  let synchronized = false;
+  if (statusImplemented) {
+    synchronized = Platform.getNtpStatus();
+  }
+
+  response.json({
+    statusImplemented,
+    synchronized,
+  });
+});
+
+SettingsController.put('/system/ntp', (request, response) => {
```

Comment:
nit: I probably would have used a POST here rather than a PUT because PUT implies you're overwriting the ntp resource with this empty content whereas POST can modify the resource in some server defined way.

Reply:
Fair enough.